app/components/chart-container.hbs

```diff
@@ -31,7 +50,7 @@
           {{svg-jar "close" class="modal__close-icon m-r-3" width="20px" height="20px"}}
         </span>
         <div class="modal-header">
-          <h3>{{t (concat 'chartTitle.' this.modalChart)}}</h3>
+          <h3>{{@location.attributes.label}} {{t (concat 'chartTitle.' this.modalChart)}}</h3>
```

Comment:
Should be `{{t (concat 'chartTitle.' this.modalChart) place=@location.attributes.label}}` for better intl support

Reply:
In non-modal form we use `{{t (concat 'chartTitle.' this.modalChart)}}` without the `place`